crates/cairo-addons/src/vm/hint_definitions.rs

```diff
@@ -0,0 +1,6 @@
+mod dict;
+pub use dict::HINTS as DICT_HINTS;
+mod hashdict;
+pub use hashdict::HINTS as HASHDICT_HINTS;
+mod utils;
+pub use utils::HINTS as UTILS_HINTS;
```

Comment:
don't know what clippy thinks nor how to enforce, but I prefer when we first have all the `mod`, then all the `(pub) use`